src/language/parser.js

```diff
@@ -70,7 +70,7 @@ import { DirectiveLocation } from './directiveLocation';
 /**
  * Configuration options to control parser behavior
  */
-export type ParseOptions = {
+export type ParseOptions = {|
```

Comment:
This may be a little dangerous, as I think there are places we do `ParseOptions & { ... }`, such as in [buildASTSchema](https://github.com/graphql/graphql-js/blob/4fd9260efa247d60f74f3ebc259c8a3c2b03af61/src/utilities/buildASTSchema.js#L503).

This would mean I can't create a generic `const REPO_OPTIONS = { ... };` that could be used everywhere I call `parse` or `buildASTSchema`

Reply:
@mjmahone Good catch 👍
I missed that.
I reverted this change and change to `buildASTSchema` options.